crates/ra_editor/src/lib.rs

```diff
@@ -31,10 +31,19 @@ pub struct HighlightedRange {
     pub tag: &'static str,
 }
 
+#[derive(Debug, Copy, Clone)]
+pub enum Severity {
+    Error,
+    Warning,
```

Comment:
Should these severities be so tied to the LSP? I think we currently only use `Error` and `Warning`. @matklad, what are your thoughts on this.